.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+
+env:
+  - ILLUMINATE_VERSION=5.5.* TESTBENCH_VERSION=3.5.* PHPUNIT_VERSION=^6.0
+  - ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
+  - ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
+
+before_install:
+  - travis_retry composer self-update
+  - travis_retry composer update --no-interaction --prefer-source
```

Comment:
This should be in your install step, before install is not meant for installing your dependencies. Also why are you passing —prefer-source, wouldn’t someone using this library be using dist versions of our dependencies. You want to replicate as close to a real world scenario as possible